bin/station.js

```diff
@@ -22,6 +22,8 @@ Sentry.init({
 
 await fs.mkdir(join(paths.moduleCache, 'saturn-L2-node'), { recursive: true })
 await fs.mkdir(join(paths.moduleState, 'saturn-L2-node'), { recursive: true })
+await fs.mkdir(join(paths.moduleCache, 'zinnia'), { recursive: true })
+await fs.mkdir(join(paths.moduleState, 'zinnia'), { recursive: true })
```

Comment:
```suggestion
for (const moduleName of ['saturn-L2-node', 'zinnia']) {
  await fs.mkdir(join(paths.moduleCache, moduleName), { recursive: true })
  await fs.mkdir(join(paths.moduleState, moduleName), { recursive: true })
}
```

I meant to refactor this once we add a second module

Reply:
Could also pull this module list into a variable that can be reused in lines 46-51